PhpcsChanged/Cli.php

```diff
@@ -128,7 +128,8 @@ function printHelp(): void {
 	printTwoColumns([
 		'--git-staged' => 'Compare the staged version to the HEAD version (this is the default).',
 		'--git-unstaged' => 'Compare the working copy version to the staged (or HEAD) version.',
-		'--git-branch <BRANCH>' => 'Compare the HEAD version to the HEAD of a different branch.',
+		'--git-branch <BRANCH>' => 'Compare the HEAD version to the HEAD of a different branch (deprecated).',
+		'--git-base-object <OBJECT>' => 'Compare the HEAD version to version found in OBJECT.',
```

Comment:
I don't feel strongly about it but would `--git-base` be a shorter and just as valid option name? Or does it leave some ambiguity that I'm not thinking of?

Reply:
Agreed. `git-base-object` is too verbose. Renamed